airflow-core/tests/unit/decorators/test_bash.py

```diff
@@ -37,7 +37,7 @@
 
 if TYPE_CHECKING:
     from airflow.models import TaskInstance
-    from airflow.operators.bash import BashOperator
+    from airflow.providers.standard.operators.bash import BashOperator
```

Comment:
Let's make it part of https://github.com/apache/airflow/pull/48683 since we should change L29 too and not import `RenderedTaskInstanceFields` for AF 3

Reply:
Alright will do